sites/friday/src/manage-resources/resource-init.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Resource initialization
+description: Learn how resources are allocated by default upon first deployment, and how you can define a resource initialization strategy that better fits your needs.
+weight: -200
+keywords:
+  - "resources"
+  - "flexible resources"
+  - "CPU"
+  - "RAM"
+  - "disk storage"
+  - "horizontal scaling"
+  - "vertical scaling"
+  - "scaling"
+---
+
+When you first deploy your {{% vendor/name %}} project, and whenever you add a new app or service,
+{{% vendor/name %}} allocates the following default resources to every container:
+
+| Resource type               | Amount |
+| --------------------------- | ----------- |
+| CPU                         | 0.5 |
+| RAM                         | Depends on the [container profile](#advanced-container-profiles). |
+| Disk size (only applicable if the app or service requires a disk)                   | 512 MB |
+
+If you don't want to use those default resources, you can define a different [resource initialization strategy](#resource-initialization-strategies).
+You can also [adjust those resources](#configure-resources) after your project or new container has been deployed.
```

Comment:
I think the link here should be `/manage-resources/adjust-resources.html` (or maybe to the `.md` extension, not sure how it works)

Reply:
Fixed